src/dao.ts

```diff
@@ -324,12 +270,31 @@ export class DAO implements IStateful<IDAOState> {
     return Stake.search(this.context, options, apolloQueryOptions)
   }
 
-  /**
-   * get (an observable of) the Ether balance of the DAO from the web3Provider
-   *
-   * @return an observable stream of BN number instances
-   */
-  public ethBalance(): Observable<BN> {
-    return this.context.ethBalance(this.id)
+  public createProposal(options: IProposalBaseCreateOptions) {
```

Comment:
This function can be simplified. Instead of doing the collection query, we can instead just query the plugin by ID since we have the dao's address and plugin's address. These two make the plugin's ID: https://github.com/daostack/subgraph/blob/327e68cbc4ee2f683d2660119ecaf4a220cace5e/src/mappings/Controller/mapping.ts#L63

Ideally the syntax would be:
```
// Plugin.instantiate(id) is a static factory method that
// instantiates derived types based on the queried data's name
const plugin = Plugin.instantiate(
  Plugin.calculateId({ daoAddress, pluginAddress })
)

plugin.createProposal(options)
```